### PR TITLE
Fix "Socketed Gems are supported by..." mods for trigger skills (e.g. CwC)

### PR DESCRIPTION
**Pull request by Nostrademous**
Fixes build: [https://pastebin.com/7EX1mcK3](https://pastebin.com/7EX1mcK3)

In that specific case "Cast while Channelling" has a primary and secondary effect. Current implementation does not add the secondary effect eliminating the trigger.

### CODE REVIEW

File: src/Modules/ModParser.lua [hunk 1]
```diff
@@ -1326,6 +1326,22 @@ local function triggerExtraSkill(name, level, noSupports)
 		}
 	end
 end
+local function supportedByEffects(name, level)
+	local skillId = gemIdLookup[name] or gemIdLookup[name:gsub("^increased ","")]
+	if skillId then
+		local gemId = data.gemForBaseName[data.skills[skillId].name .. " Support"]
+		if gemId then
```
Comment (Wires77) [inline src/Modules/ModParser.lua:1333]: These can be in-lined.

```suggestion
	if gemIdLookup[name] or gemIdLookup[name:gsub("^increased ","")] then
		if data.gemForBaseName[data.skills[skillId].name .. " Support"] then
```

**Review by Wires77 — CHANGES_REQUESTED** [review]
Since this line is the only place where this new function is used, lets in-line it like the surrounding code to match the rest of the file's style

File: src/Modules/ModParser.lua (hunk 1 above)
Comment (Nostrademous) [reply src/Modules/ModParser.lua:1333]: @Wires77 the variable `skillId` is used on line 1331. You deleted the variable when in-lining.

**Review by Wires77 — APPROVED** [review]
LGTM

**Pull request merged**